internal/dinosaur/pkg/services/util_test.go

```diff
@@ -352,3 +353,40 @@ func Test_contains(t *testing.T) {
 		})
 	}
 }
+
+func Test_formatNamespace(t *testing.T) {
+	type testCase struct {
+		id        string
+		namespace string
+		error     bool
```

Comment:
I prefer being explicit with properties in test. In this case for example not obvious which of these properties is the expected result.

```suggestion
		expectError     bool
```

Reply:
Renamed as suggested.